config/config.exs

```diff
@@ -1,6 +1,6 @@
 # This file is responsible for configuring your application
 # and its dependencies with the aid of the Mix.Config module.
-use Mix.Config
+import Config
```

Comment:
Seems unrelated to the change being proposed, you could probably remove these to make the changeset smaller

Reply:
I have reverted them to make the changeset smaller. @kdawgwilk could you review again?

Reply:
My approval won't count unfortunately since I am not a maintainer